### PR TITLE
First draft of Trusted Web Activities post

### PR DESCRIPTION
**Pull request by sbracegirdle**

### CODE REVIEW

File: _posts/2020-06-11-trusted-web-activities.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+At this point it may take a few or several days before the app is published to the store. Later updates are not required, as the TWA will always serve the latest version of your app from the web.
+
+Once your app is published, you may need to repeat the steps in the asset link section, as the published APK will have a different signature. After that, your app will be ready to be installed by your users.
+
+
+### Conclusion
+
+We've learned what a TWA is, how we can create one with PWABuilder and how to publish it to the Play Store.
+
+This is a big step forward for publication of web apps to the native app stores of the popular mobile platforms, as it eliminates the complexity of creating a native wrapper. I'm hoping that this sets a precedent that Apple will soon follow with their iOS App Store.
```
Comment (temyers) [inline _posts/2020-06-11-trusted-web-activities.md:130]: This is a big part for me - not just the how, but the why.

Make a big deal of this in your lead in and perhaps expand a little on why this is awesome.

Why is it better than building native apps?  etc.

File: _posts/2020-06-11-trusted-web-activities.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+To submit your app for review, you need to [roll out a release](https://support.google.com/googleplay/android-developer/answer/7159011#rollout) under *App releases*. Walk through the steps of the wizard, select *Review*, *Review and roll out release* and then *Confirm* the rollout.
+
+At this point it may take a few or several days before the app is published to the store. Later updates are not required, as the TWA will always serve the latest version of your app from the web.
+
+Once your app is published, you may need to repeat the steps in the asset link section, as the published APK will have a different signature. After that, your app will be ready to be installed by your users.
+
+
+### Conclusion
+
+We've learned what a TWA is, how we can create one with PWABuilder and how to publish it to the Play Store.
```
Comment (temyers) [inline _posts/2020-06-11-trusted-web-activities.md:128]: I'm a little confused by here - it seems to be half way between a tutorial and an overview.  

Can you improve the signposting between each step?
Can you follow a single worked example (or make it explicit)?

File: _posts/2020-06-11-trusted-web-activities.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+This is a big step forward for publication of web apps to the native app stores of the popular mobile platforms, as it eliminates the complexity of creating a native wrapper. I'm hoping that this sets a precedent that Apple will soon follow with their iOS App Store.
+
+If you need help with building Progressive Web Apps, or TWAs, please [get in touch with us](https://mechanicalrock.io/contact).
+
```
Comment (temyers) [inline _posts/2020-06-11-trusted-web-activities.md:133]: The structure doesn't seem to flow quite right - pretty much everything is `h3`
maybe something like
```
<lead in>
lots of opinions here - hook the reader to learn more...
## Setup
blah
### TWAs
### PWAs

## Worked example
### step 1
### step 2
...
## Conclusion
wrap up
```

File: _posts/2020-06-11-trusted-web-activities.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+You don't need to make any drastic changes to your existing web app. You also don't need to install native development tools such as Android Studio, or write any native code.
+
+For example, let's say you have built an app that works well on mobile devices due to responsive web design. You can create an Android Package for that app via [PWA Builder](https://pwabuilder.com) and then publish that app to the Google Play Store.
+
+You have been able to integrate web apps with native apps for a while, but doing so has always been arduous. The previous approach was to create a wrapping native app and deal with the complexities of updating it separately to your web app.
+
+TWA's solve this problem by serving your app from the web with an actual browser, but without the address bar at the top or status bar at the bottom. This is like adding web app to your home screen from the in-browser feature, but with some differences that we will soon discover.
+
+### What is a Progressive Web App (PWA)?
```
Comment (temyers) [inline _posts/2020-06-11-trusted-web-activities.md:25]: Look to link to our PWA capability report.

https://www.mechanicalrock.io/docs/case-studies/pwa-capability-report.pdf

File: _posts/2020-06-11-trusted-web-activities.md (hunk 2 above)
Comment (sbracegirdle) [reply _posts/2020-06-11-trusted-web-activities.md:128]: I've made an attempt to improve the flow and add sign-posting. Do you think the worked example is clearer now?

File: _posts/2020-06-11-trusted-web-activities.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+In this post we'll learn about a new feature in Android called Trusted Web Activities, which enables publishing of web apps to the Google Play Store. We'll also work through an example of setting one up for an existing web app.
+
+Let's get started!
+
+## Background
+
+### What is a Trusted Web Activity?
+
+In the introduction I mentioned Trusted Web Activities, so what are they?
```
Comment (temyers) [inline _posts/2020-06-11-trusted-web-activities.md:29]: >In the introduction I mentioned Trusted Web Activities, so what are they?

Unnecessary - suggest delete.

Your intro is short, and you describe what they are in the next para.

File: _posts/2020-06-11-trusted-web-activities.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+* A [Service Worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) that helps provide offline support and caching. You can use tools such as workbox to help with this.
+* Usage of modern web capabilities such as the [Push API](https://developer.mozilla.org/en-US/docs/Web/API/Push_API) and [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API) to provide user experiences that were previously exclusive to native apps.
+* A [Web App Manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest) to provide additional metadata to consuming platforms.
+* [Responsive Web Design](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design) to ensure a good user experience across a variety of devices.
+
+This makes PWAs well suited for integration with native mobile platforms via features such as TWAs.
+
+For a more information on PWAs, I highly recommend [PWA: Cache me if you can](https://mechanicalrock.github.io/2020/02/25/pwa-cache-me-if-you-can.html) and our [PWA Capability Report](https://www.mechanicalrock.io/docs/case-studies/pwa-capability-report.pdf).
```
Comment (temyers) [inline _posts/2020-06-11-trusted-web-activities.md:54]: Like!

**Pull request merged**